api/namex/resources/name_requests/api_models.py

```diff
@@ -33,7 +33,8 @@
     'designation': fields.String('Name designation based on entity type'),
     'conflict1_num': fields.String('The corp_num of the matching name'),
     'conflict1': fields.String('The matching corp name'),
-    'consent_words': fields.Nested(consent_model)
+    'consent_words': fields.Nested(consent_model),
+    'corpNum': fields.String('Corp Num')
```

Comment:
what about  consumptionDate

Reply:
Not sure about consumptionDate... I didn't see that in the ticket.

Reply:
consumptionDate and corpNum define this as a consumed name.